### ISSUE TITLE
[Bug][PHP] Removal of required ';' char on minification

### ISSUE THREAD

**Issue by S-K-Tiger** [Josee9988/MinifyAllCli#72]
# **🐞 Bug Report**

## **Describe the bug**
PHP minification may create invalid syntax. When a ';' is directly in-front of a '}' the ';' is removed creating a syntax error.

---

### **Is this a regression?**
Not as far as I'm aware

---
### **To Reproduce**

Minify:
```php
<?php
if(true){$var=1;$var2=2;}
if(true){$var=1;$var2=2; }
?>
```

---
### **Expected behaviour**
```php
<?php
if(true){$var=1;$var2=2;}
if(true){$var=1;$var2=2;}
?>
```
or
```php
<?php $var=1;$var2=2;?>
```

---
### **Observed behaviour**
```php
<?php
if(true){$var=1;$var2=2}
if(true){$var=1;$var2=2;}
?>
```

---
### **Your environment**
* Plugin version: v2.10.0

**Comment by Josee9988** [comment]
Hi @S-K-Tiger
I'm sorry but the Minifier doesn't work with raw PHP code. It just supports .php files to allow html minimization inside .php files.
So that PHP code is treated like "html" and that's why you might find inconsistencies in your code.
You have to minify your html code manually using the "Mininify Selected Text" option.

TL;DR
The extension doesn't minify PHP, in this case, your PHP code is being treated as HTML and the extension allows the minimization of HTML code in .php files. You should not minify your PHP code with MinifyAll as this is not currently supported!
